chroma-dependencies/Makefile

```diff
@@ -21,7 +21,7 @@ cleanrepo: TARGET=cleanrepo
 requirements::
 	# these are the ones provided by EL and EPEL and our own repo
 	set -e;                                                   \
-	{ echo "../chroma-externals/netaddr-0.7.5.zip";           \
+	{ echo "https://github.com/drkjam/netaddr/archive/rel-0.7.5.tar.gz"; \
```

Comment:
just trying to be consistent with your review approach, I don't think we should be introducing double standards. if you recommend a format you should stick to it yourself